source/chapters/advanced_topics.rst

```diff
@@ -106,6 +106,2785 @@ send messages to text displays on the controller.
+
+   :range: binary
+   :feedback: If VC isn't enabled on any decks, enable it on the first one we're receiving samples for. If VC is enabled on a single (exclusive) deck, and another deck is setup to receive samples, disable it on the former deck and enable it on the next eligible deck (ordered by deck number). If VC is enabled on multiple decks, don't do anything.
+
+   .. versionadded:: v1.10.0
+
+
+The ``[MicrophoneN]`` group
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below, N=2 up to the number of active microphones. e.g ``[Microphone2]``. For Microphone 1, just use ``[Microphone]``.
+
+.. mixxx:control:: [MicrophoneN],enabled
+
+   1 if a microphone input is enabled, 0 if not.
+
+   :range: binary
+   :feedback: Microphone is enabled.
+
+   .. versionadded:: v1.10.0
+
+
+.. mixxx:control:: [MicrophoneN],orientation
+
+   Set microphone orientation,
+
+   :range:
+      ===== ===================================
+      Value Meaning
+      ===== ===================================
+      0     Left side of crossfader
+      1     Center (not affected by crossfader)
+      2     Right side of crossfader
+      ===== ===================================
+   :feedback: N/A
+
+   .. versionadded:: v1.10.0
+
+
+.. mixxx:control:: [MicrophoneN],PeakIndicator
+
+   Indicates when the signal is clipping (too loud for the hardware and is being distorted)
+
+   :range: binary
+   :feedback: Microphone Clip light
+
+   .. versionadded:: v1.10.0
+
+
+.. mixxx:control:: [MicrophoneN],talkover
+
+   Hold value at 1 to mix microphone input into the master output.
+
+   :range: binary
+   :feedback: Talk button
+
+   .. versionadded:: v1.10.0
+
+
+.. mixxx:control:: [MicrophoneN],volume
+
+   Adjusts the microphone volume fader
+
+   :range: default
+   :feedback: Microphone volume fader changes
+
+   .. versionadded:: v1.10.0
+
+
+.. mixxx:control:: [MicrophoneN],pregain
+
+   Adjusts the gain of the mic input
+
+   :range: 0.0..1.0..4.0
+   :feedback: Microphone gain knob
+
+
+.. mixxx:control:: [MicrophoneN],mute
+
+   Mutes the channel
+
+   :range: binary
+   :feedback: Mute button
+
+   .. versionadded:: v2.0.0
+
+
+.. mixxx:control:: [MicrophoneN],VuMeter
+
+   Outputs the current instantaneous microphone volume
+
+   :range: default
+   :feedback: Microphone VU meter changes
+
+   .. versionadded:: v1.10.0
+
+
+The ``[Recording]`` controls
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. mixxx:control:: [Recording],toggle_recording
+
+   Turns recording on or off.
+
+   :range: binary
+   :feedback: Recording icon
+
+
+.. mixxx:control:: [Recording],status
+
+   Indicates whether Mixxx is currently recording.
+
+   :range:
+      ===== ====================
+      Value Meaning
+      ===== ====================
+      0     Recording Stopped
+      1     Initialize Recording
+      2     Recording Active
+      ===== ====================
+   :feedback: Recording icon
+
+
+The ``[AutoDJ]`` controls
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Maybe add a note/link to the [Library],AutoDJ... controls ?
I know this is just the inital mvoe to the manual, don't wanna demand too much ;)

Reply:
Done.